glossary.md

```diff
@@ -0,0 +1,14 @@
+# Glossary
+
+### APY
+
+Annual Percentage Yield, is the normalized representation of an interest rate,
+based on a compounding period over one year. Note that APYs provided are rather
+ballpark level indicators and not so much precise future results.
+
+### BCV
```

Comment:
Just starting with the basics. We can add more in subsequent pull requests. Hope the content is accurate.

Reply:
I suggest we add a glossary for APR too. Although we do not use this term in Olympus, I think adding a definition allows readers to understand the difference between APR and APY.

Suggestion:
Annual Percentage Rate refers to annualized interest rate **without** taking into account the effect of compounding.

Reply:
I added APR. Good call.